Client/Frontend/Browser/TopTabsViewController.swift

```diff
@@ -170,8 +170,13 @@ class TopTabsViewController: UIViewController {
     
     func privateTabTapped() {
         delegate?.topTabsDidPressPrivateTab()
-        self.collectionView.reloadData()
-        scrollToCurrentTab(false)
+        UIView.animateWithDuration(0, animations: {
```

Comment:
So the reason I initially used the 0 duration animation block was because the collectionview must be fully finished loading before scrollRectToVisible can be called. But I looked into it further and scrollToItemAtIndexPath can be called directly after reloadData() so I switched it to that :)